app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -178,11 +177,21 @@ class LeanbackChannelWorker(
 
 		// Poster image
 		setPosterArtAspectRatio(WatchNextPrograms.ASPECT_RATIO_16_9)
-		setPosterArtUri(Uri.parse(apiClient.GetImageUrl(item, ImageOptions().apply {
-			format = ImageFormat.Png
-			height = 288
-			width = 512
-		})))
+		if (item.parentThumbItemId != null) {
+			setPosterArtUri(Uri.parse(apiClient.GetImageUrl(item.seriesId, ImageOptions().apply {
+				format = ImageFormat.Png
+				height = 288
+				width = 512
+				imageType = org.jellyfin.apiclient.model.entities.ImageType.Thumb
+			})))
+		}
+		else {
+			setPosterArtUri(Uri.parse(apiClient.GetImageUrl(item, ImageOptions().apply {
+				format = ImageFormat.Png
+				height = 288
+				width = 512
+			})))
+		}
```

Comment:
The "next up" row displays singular entries, not shows. It should use the thumbnail instead of poster.

Reply:
Right now the Next Up row inside the app displays the thumbnail for the series if there is one, and the thumbnail for the episode if there isn’t. This would be to match that behavior

Reply:
I agree that we should make this behavior consistent but it's something that needs to change in the home screen, not the next up row.